src/thunderbots/software/ai/hl/stp/evaluation/team.cpp

```diff
@@ -0,0 +1,20 @@
+#include "ai/hl/stp/evaluation/team.h"
+
+
+Robot Evaluation::nearest_friendly(const Team friendly_team, const Point ref_point)
+{
+    unsigned nearestRobotId = friendly_team.getAllRobots()[0].id();
```

Comment:
hmm, what if the team is empty? I'm genuinely unsure of what the best way to handle this case would be. The two options I can think of would be to return an optional (https://en.cppreference.com/w/cpp/utility/optional) or throw an exception. @MathewMacDougall what do you think? We're gonna see this come up a lot, so it might be good to sit down with you and @matthewberends and figure out what we want to do in these cases?

Reply:
Lots of the functions in Team class seem to use optionals (ex: getRobotById) so having a team with no robots seems like a valid state. With that in mind I think we should return an optional for this evaluation function.

Reply:
Ah, nice job looking into that! Sounds good to me!

Reply:
I agree with using optional here :+1: